plugins/commitlint/readme.md

```diff
@@ -0,0 +1,20 @@
+# @dotcom-tool-kit/commitlint
+
+Tool Kit plugin to run [commitlint](https://commitlint.js.org/)
+
+## Installation & Usage
+
+With Tool Kit [already set up](https://github.com/financial-times/dotcom-tool-kit#installing-and-using-tool-kit), install this plugin as a dev dependency:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/commitlint
+```
+
+And add it to your repo's `.toolkitrc.yml`:
+
+```yml
+plugins:
+  - '@dotcom-tool-kit/commitlint'
+```
+
+By default the plugin will run on the `git:commitmsg` command. There are no further configurations in Tool Kit, [use one of the config file locations that Commitlint supports](https://commitlint.js.org/reference/configuration).
```

Comment:
**suggestion**: although the task has no options, you should still add an [empty schema](https://github.com/Financial-Times/dotcom-tool-kit/blob/main/lib/schemas/src/tasks.ts#L23) in `schemas/tasks.ts` with a description for the task, add `<!-- begin autogenerated docs -->` to this readme, and run `scripts/generate-docs.js`

Reply:
Been faffing around with this and can't seem to get it to generate anything. I have the empty schema and nothing is changing when I generate docs. I tried logging stuff and it seems like `formatPluginSchemas` isn't returning anything

Reply:
Oh nevermind need to add it to the task